Content.Server/Maps/GameMapManager.cs

```diff
@@ -190,10 +194,17 @@ public bool CheckMapExists(string gameMap)
 
     private bool IsMapEligible(GameMapPrototype map)
     {
-        return map.MaxPlayers >= _playerManager.PlayerCount &&
-               map.MinPlayers <= _playerManager.PlayerCount &&
+        // return map.MaxPlayers >= _playerManager.PlayerCount &&
+        //        map.MinPlayers <= _playerManager.PlayerCount &&
+        //        map.Conditions.All(x => x.Check(map)) &&
+        //        _entityManager.System<GameTicker>().IsMapEligible(map);
+        // Harmony Queue Start
+        // Modified to make merging easier
+        return map.MaxPlayers >= CurrentPlayerCount &&
+               map.MinPlayers <= CurrentPlayerCount &&
                map.Conditions.All(x => x.Check(map)) &&
                _entityManager.System<GameTicker>().IsMapEligible(map);
+        // Harmony Queue End
     }
```

Comment:
it seems like we dont need any of the changes present in this file. This has strictly to do with selecting the next incoming map, and ensuring the map loader is selecting a map of the appropriate size by not counting the number of players in the queue as being active players in the next round.